cg/meta/workflow/rnafusion.py

```diff
@@ -93,6 +89,19 @@ def build_samplesheet_content(
         }
         return samplesheet_content
 
+    def create_samplesheet_csv(
+        self,
+        samplesheet_content: Dict[str, List[str]],
+        headers: List[str],
+        config_path: Path,
+    ) -> None:
+        """Write sample sheet csv file."""
+        with open(config_path, "w") as outfile:
+            outfile.write(",".join(headers))
+            for i in range(len(samplesheet_content[NFX_SAMPLE_HEADER])):
+                outfile.write("\n")
+                outfile.write(",".join([samplesheet_content[k][i] for k in headers]))
```

Comment:
We have a csv write controller in io that we should use here

Reply:
All the sample sheets methods need to be cleaned up. They are quite messy. I was hoping to get away with them on this PR. But FINE I can do it here 😬

Reply:
oh I hadn't read properly 😶‍🌫️  leaving this for a following PR